addon/models/goal.js

```diff
@@ -22,8 +25,13 @@ export default Activity.extend({
   description: attr('string'),
   progress: attr('string'),
   updateText: attr('string'),
-  product_owner: attr('number'),
-  head_design: attr('number'),
-  head_frontend: attr('number'),
-  head_backend: attr('number')
-});
\ No newline at end of file
+
+  contributers: hasMany('user'),
+
+  product_owner: belongsTo('user', { inverse: null }),
+  head_design: belongsTo('user', { inverse: null }),
+  head_frontend: belongsTo('user', { inverse: null }),
+  head_backend: belongsTo('user', { inverse: null }),
+
+  nr_volunteers: filterBy('user').length
```

Comment:
How does this work? What does it filter, the list of all the goals? Also, this should probably not be a number but just a list of volunteers, because then the length will update when the list updates.